csrankings-p.csv

```diff
@@ -88,6 +88,7 @@ Panayotis G. Cottis,NTUA,https://www.ece.ntua.gr/en/staff/19,NOSCHOLARPAGE
 Panayotis Mavromatis,New York University,http://steinhardt.nyu.edu/faculty/Panayotis_Mavromatis,2ECioBQAAAAJ
 Panayotis Tsanakas,NTUA,http://www.cslab.ntua.gr/~panag,UdV8-qIAAAAJ
 Panfeng Huang,NWPU,https://teacher.nwpu.edu.cn/m/en/2005000156.html,NOSCHOLARPAGE
+Pang Wei Koh,University of Washington,https://koh.pw/

```

Comment:
Missing Google Scholar ID. Please verify you will be full-time at UW.

Reply:
Sorry for missing the Google Scholar ID. I've added it in now. I'm currently an affiliate at UW and will be full-time next Fall. Thank you!